Tests/RouterTest.php

```diff
@@ -310,7 +228,7 @@ public static function seedTestParseRoute()
 			array('', false, array('controller' => 'DefaultController', 'vars' => array()), true),
 			array('login', false, array('controller' => 'LoginController', 'vars' => array()), true),
 			array('articles', false, array('controller' => 'ArticlesController', 'vars' => array()), true),
-			array('articles/4', false, array('controller' => 'ArticleController', 'vars' => array('article_id' => 4)), true),
+			array('articles/4', false, array('controller' => 'ArticleController', 'vars' => array('article_id' => '4')), true),
```

Comment:
Why are these no longer `int`'s?

Reply:
The controller doesn't typecast values so doing `assertSame` fails because of string versus int.

Reply:
Nevermind, re-checked and it's working as int.